packages/eslint-config-sdk/src/index.js

```diff
@@ -173,6 +174,8 @@ module.exports = {
         '@typescript-eslint/explicit-member-accessibility': 'off',
         '@typescript-eslint/no-explicit-any': 'off',
         '@typescript-eslint/no-non-null-assertion': 'off',
+        '@typescript-eslint/no-empty-function': 'off',
```

Comment:
I'm curious - why turn `no-empty-function` off?

Reply:
Feel like it's not valuable for tests, where you might want to quickly add empty functions if you are stubbing out a class/object.